src/senselab/utils/data_structures/__init__.py

```diff
@@ -12,9 +11,15 @@
     SenselabModel,
     SentenceTransformersModel,
     SpeechBrainModel,
-    TorchAudioModel,
     TorchModel,
     check_hf_repo_exists,
 )
+
+# Conditionally import TorchAudioModel only if torchaudio is available
+try:
+    from .model import TorchAudioModel
+except ImportError:
```

Comment:
Actually this one should remain `ImportError` , because `.model` IS found, but has an error upon importing, so it uses the more generic error class.